etc/profile-a-l/electron.profile

```diff
@@ -4,6 +4,8 @@
 # Persistent local customizations
 include electron.local
 
+whitelist ${HOME}/.config/electron-flag*.conf
+whitelist ${HOME}/.config/Electron
```

Comment:
Move down to the other whitelists.

Also shouldn't we `noblacklist` them here?

Reply:
Added the `noblacklist`s, thanks for reminding me about those!

> Move down to the other whitelists.
Has this changed recently? As the blacklisting is done in `disable-programs.inc` I added them above the block that includes that file...